packages/admin-panel/src/table/DataFetchingTable/DataFetchingTable.jsx

```diff
@@ -37,32 +34,38 @@ import {
 } from '../actions';
 import { FilterCell } from './FilterCell';
 import { Pagination } from './Pagination';
+import { DisplayCell, HeaderDisplayCell } from './Cells';
 
-const BUTTON_COLUMN_WIDTH = '4.5rem';
-
-const Cell = styled(TableCell)`
-  vertical-align: middle;
-  font-size: 0.75rem;
-  padding: 0.7rem;
-  overflow: hidden;
-  max-width: ${({ $isButtonColumn }) => ($isButtonColumn ? BUTTON_COLUMN_WIDTH : '0')};
-  width: ${({ $isButtonColumn }) => ($isButtonColumn ? BUTTON_COLUMN_WIDTH : 'auto')};
-  white-space: nowrap;
-  text-overflow: ellipsis;
+const ErrorAlert = styled(Alert).attrs({
+  severity: 'error',
+})`
+  margin: 0.5rem;
 `;
 
-const HeaderCell = styled(Cell)`
-  color: ${({ theme }) => theme.palette.text.secondary};
-  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
-  .MuiTableSortLabel-icon {
-    opacity: 1;
+const TableContainer = styled(MuiTableContainer)`
+  position: relative;
+  flex: 1;
+  overflow: auto;
+  table {
+    min-width: 45rem;
+  }
+  thead {
+    position: sticky;
```

Comment:
Huh, does this not come with the `stickyHeader` prop? 🧐

Reply:
Only the `Table` component does, but in this case we need to add some extra stuff to make the whole `thead` sticky as we have 2 rows to be sticky now (header and filter row)

Reply:
Ah! In that case I think https://github.com/beyondessential/tupaia/pull/5610#discussion_r1590421881 should also be ignored

Thanks for clarifying 🙏